pkgs/development/python-modules/dash/default.nix

```
@@ -65,12 +64,12 @@ buildPythonPackage rec {
 
     export HOME=$(mktemp -d)
 
-    yarn config --offline set yarn-offline-mirror ${yarnDeps}
+    yarn config --offline set yarn-offline-mirror $yarnOfflineCache
```

```suggestion
    yarn config --offline set yarn-offline-mirror ${yarnOfflineCache}
```

`yarnOfflineCache` is not considered as environment variable with `__structuredAttrs` enabled.

---

Thanks for the heads up
Didn't know about `__structuredAttrs`